components/Author.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import AuthorCollection from '../interfaces/AuthorCollection'
+import { MarkdownFileWithUrl } from '../pages/authors'
+
+import ContentCardList from './ContentCardList'
+import PageLayout from './PageLayout'
+
+export interface Props {
+ author?: AuthorCollection
+ records: MarkdownFileWithUrl[]
+}
+
+const Author: React.FunctionComponent<Props> = props => {
+  const authorName = props.author?.name || 'Unnamed author'
+  const authorImage = props.author?.image
```

Comment:
With the suggested change to make author non-optional, we can rely on the fact that these values are guaranteed to be present.
```suggestion
  const authorName = props.author.name
  const authorImage = props.author.image
```